orchestration/templates/run-extraction-pipeline.yaml

```diff
@@ -0,0 +1,38 @@
+{{- $version := default "latest" .Chart.AppVersion }}
+  {{- with .Values.dataflow }}
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: run-extraction-pipeline
+spec:
+  templates:
+    - name: main
+      inputs:
+        parameters:
+          - name: gcs-bucket
+          - name: output-prefix
+      container:
+        image: us.gcr.io/broad-dsp-gcr-public/encode-extraction:{{ $version }}
+        command: []
+        args:
+          - --runner=dataflow
+          {{- $bucket := "{{inputs.parameters.gcs-bucket}}" }}
+             - --inputPrefix=gs://{{ $bucket }}/{{ "{{inputs.parameters.input-prefix}}" }}
+          - --outputPrefix=gs://{{ $bucket }}/{{ "{{inputs.parameters.output-prefix}}" }}
```

Comment:
Double-check that this is the actual arg expected by the pipeline. I remember that the two are inconsistent; one uses "prefix" and the other uses "path", I think.